assets/js/pluginsidebar/components/sidebar/index.js

```diff
@@ -216,20 +220,22 @@ class Sidebar extends React.PureComponent {
         name="publish-to-apple-news"
         title={__('Publish to Apple News Options', 'apple-news')}
       >
-        <div className="components-panel__body is-opened">
+        <div
+          className="components-panel__body is-opened"
+          id="apple-news-publish"
+        >
+          <Notifications />
           <h3>Sections</h3>
-          {
-            sections.map(({ id, name }) => (
-              <CheckboxControl
-                key={id}
-                label={name}
-                checked={- 1 !== selectedSectionsArray.indexOf(id)}
-                onChange={
-                  (checked) => this.updateSelectedSections(checked, id)
-                }
-              />
-            ))
-          }
+          {Array.isArray(sections) && sections.map(({ id, name }) => (
```

Comment:
Would this make sense as an unordered list?

Reply:
Here, it probably does, yes.